docs/feature_encoders.md

```diff
@@ -33,27 +45,31 @@ If you are using different pinouts for the encoders on each half of a split keyb
 
 The callback functions can be inserted into your `<keyboard>.c`:
 
-    void encoder_update_kb(uint8_t index, bool clockwise) {
-        encoder_update_user(index, clockwise);
-    }
+```c
+void encoder_update_kb(uint8_t index, bool clockwise) {
+    encoder_update_user(index, clockwise);
+}
+```
 
 or `keymap.c`:
 
-    void encoder_update_user(uint8_t index, bool clockwise) {
-      if (index == 0) { /* First encoder */
-        if (clockwise) {
-          tap_code(KC_PGDN);
-        } else {
-          tap_code(KC_PGUP);
-        }
-      } else if (index == 1) { /* Second encoder */  
-        if (clockwise) {
-          tap_code(KC_UP);
-        } else {
-          tap_code(KC_DOWN);
-        }
-      }
+```c
+void encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) { /* First encoder */
+    if (clockwise) {
```

Comment:
```suggestion
    if (clockwise == ENCODER_CLOCKWISE) {
```

?

Reply:
I mean, yeah, that's a possibility, but that seems a lot less elegant and much more convoluted.  It's a boolean, is it clockwise, or not.    If it was `if (direction == ENCODER_CLOCKWISE)`, I think that would make more sense and I would be more open to it.   But that's a much larger change.

Reply:
I'm not sure I understand... when `ENCODER_FLIP` is defined, `clockwise` will be `true` if the encoder is rotated *counter-clockwise*. Am I reading this right?

Reply:
Because not all of the encoders are configured correctly.  Eg, the A and B pins may be swapped, so the directions would be wrong. 

This allows for an easier way to swap them than undefining and redefining the pins

Reply:
Ah, so this PR is about *correcting* `clockwise` if it's inverted? Got it.

Reply:
Correct. Sorry for not being clear about that!